masterlist.yaml

```diff
@@ -10362,7 +10371,8 @@ plugins:
         condition: 'active("Complete Crafting Overhaul_Remastered.esp") and not active("Hothtrooper44_ArmorCompilation_CCOR_Patch.esp")'
     tag:
       - Delev
-      - Relev
```

Comment:
It was probably unintended behaviour in bash.
But having a Relev tag used to carry forward changes to Level List Flags.
I think that's why some mods like this and Trade & Barter had them included.

Reply:
Yeah, I figured it had something to do with that. It's not mentioned in the documentation, so I assumed it didn't still do that. From testing with Trade & Barter, it looks like it still does. Not only that, but without the Relev tag, WB undoes changes to the flags even if it's the only plugin editing the leveled list.

![conflict](https://user-images.githubusercontent.com/69141501/95004430-d23bb780-059f-11eb-8299-db0b96e62c4b.png)

I'll reassess my Relev removals and open another PR in a bit. I'll also bring it up in the WB discord as it looks like this is either all unintended behavior or there's only some unintended behavior and the docs need to be updated.

Reply:
> I'll reassess my Relev removals and open another PR in a bit. I'll also bring it up in the WB discord as it looks like this is either all unintended behavior or there's only some unintended behavior and the docs need to be updated.

It would be simpler if the flags, chance and global were handled separately.
Adding Relev as a work around is going to affect the overall output of both Delev and Relev.

Reply:
Okay, I'll leave it as is and bring it up on the WB discord.

Reply:
From @Infernio:

> Yes, the leveled list patcher has really stupid flag handling logic. I want to rewrite the whole patcher eventually, so adding more tags onto the broken mess that it is doesn't make sense right now.

Reply:
> > Yes, the leveled list patcher has really stupid flag handling logic. I want to rewrite the whole patcher eventually, so adding more tags onto the broken mess that it is doesn't make sense right now.

@sibir-ine @Infernio 
We can restore the `Relev` tags for now, with a comment to note why.
And review it if how flags are handled is ever changed in the leveled list patcher.

Reply:
Got it, I'll review my Relev and Invent changes and open a PR some time today.